src/common/store/configure-store.js

```diff
@@ -2,24 +2,36 @@ import { createStore, compose, applyMiddleware } from 'redux';
 import { routerMiddleware } from 'react-router-redux';
 import createLogger from 'redux-logger';
 import createSagaMiddleware from 'redux-saga';
+import thunk from 'redux-thunk';
```

Comment:
I know talking about thunk vs sagas might be a heated discussion but we should probably think about keeping only one of these...
I admit this project has got sagas only cause I started with a template I created with saga when I wanted to check how it worked. 
Having said so, I'm also leaning a bit more towards saga but I can see how it looks complicated and difficult. I just wanted to give it a go and see if I could write generators in a neat way as I went though a similar journey when I switched from callbacks to promises.
Anyway I'm happy either ways ;-)

Reply:
Cool lets refac this to use saga until we sic of it. I'd rather expand my knowledge than have my way